### DISCUSSION TITLE
vdc sync

### DISCUSSION THREAD

**Discussion by ivannikita** [bb-Ricardo/netbox-sync#222] [General]
Hi, we need sync vdс tenant from vmware cloud, for create  tenant in netbox and put vm to this tenant. How we can do that?

**Comment by bb-Ricardo** [comment]
Hi, I 'm really sorry but don't really understand the scenario and the question. Can you describe this more detailed please? Thank you

**Comment by ivannikita** [comment]
Hi, Ricardo!
I would like to clarify one issue. If I understand correctly your script sorts VMs to tenants parsing VM name? We have many clients and we need to sort VMs according to the organizations where they placed.

**Reply by bb-Ricardo** [reply]
Like in folders? and you want to assign tenants based on the folder the VM is in?
